src/app/features/tracking/tracking.component.less

```
@@ -88,12 +104,34 @@
     }
   }
 }
+
+.recive__div {
```

```suggestion
.receive__div {
```
Look for the html file that uses this class and update it too.